src/pages/signin/SignInPageLayout/Footer.tsx

```diff
@@ -138,19 +142,21 @@ const columns = ({navigateFocus}) => [
     },
 ];
 
-function Footer(props) {
+function Footer({shouldShowSmallScreen = false, navigateFocus}: FooterProps) {
     const theme = useTheme();
     const styles = useThemeStyles();
     const StyleUtils = useStyleUtils();
-    const isVertical = props.shouldShowSmallScreen;
+    const {translate} = useLocalize();
+    const {isMediumScreenWidth} = useWindowDimensions();
+    const isVertical = shouldShowSmallScreen;
     const imageDirection = isVertical ? styles.flexRow : styles.flexColumn;
     const imageStyle = isVertical ? styles.pr0 : styles.alignSelfCenter;
     const columnDirection = isVertical ? styles.flexColumn : styles.flexRow;
     const pageFooterWrapper = [styles.footerWrapper, imageDirection, imageStyle, isVertical ? styles.pl10 : {}];
     const footerColumns = [styles.footerColumnsContainer, columnDirection];
-    const footerColumn = isVertical ? [styles.p4] : [styles.p4, props.isMediumScreenWidth ? styles.w50 : styles.w25];
+    const footerColumn = isVertical ? [styles.p4] : [styles.p4, isMediumScreenWidth ? styles.w50 : styles.w25];
     const footerWrapper = isVertical ? [StyleUtils.getBackgroundColorStyle(theme.signInPage), styles.overflowHidden] : [];
-
+    const getTextLinkStyle = (hovered: boolean) => [styles.footerRow, hovered ? styles.textBlue : {}];
```

Comment:
```suggestion
    const getTextLinkStyle: (hovered: boolean) => StyleProp<TextStyle> = (hovered) => [styles.footerRow, hovered && styles.textBlue];
```

The signature of this function does not look correct. Either define the function type or just remove it

Reply:
Yes that's the expected results. react-native accepts falsy values

https://github.com/facebook/react-native/blob/68e6a94538c3eb35de5477bc0fb6649514c1c6fc/packages/react-native/Libraries/StyleSheet/StyleSheet.d.ts#L22-L26